### PR TITLE
Live through nominatim service being unavailable

### PR DESCRIPTION
**Pull request by jfberry**
A reasonably common operational problem is the unavailability of the nominatim server. This falls back to an 'unknown' text if not available

## Description

Simply catch the error

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/controllers/controller.js [hunk 1]
```diff
@@ -133,7 +133,7 @@ class Controller {
 			geoCache.save(true)
 			return result
 		} catch (err) {
-			throw { source: 'getAddress', error: err }
+			return { addr: 'Unknown', flag: '' }
```
Comment (lenisko) [inline src/controllers/controller.js:136]: Going through with notification is very good idea but completely silencing error is not.
How about adding this to error log?
`this.log.error('getAddress: failed to fetch data', err)`

Comment (jfberry) [reply src/controllers/controller.js:136]: I was thinking about the permanent 'not going to install nominatim' situation and log bloat - but you're right this should perhaps be through a config.  I will add a log and do something different for a 'none' situation

Comment (LucianoNeo) [reply src/controllers/controller.js:136]: Working like a charm, thank you!

**Comment by jfberry** [comment]
Variant pushed

**Pull request closed without merging**